Model/Runner.php

```diff
@@ -35,24 +35,29 @@ public function __construct(
         ];
     }
 
-    public function run($diff)
+    public function run($line)
     {
-        foreach ($diff as $line) {
-            $pathInfo = $this->lineProcessor->toPathInfo($line);
+        $result = [];
+        $result['type'] = '';
+        $result['items'] = [];
+        $result['path'] = '';
```

Comment:
https://github.com/sdinteractive/Magento2_SomethingDigital_UpgradeHelper/blob/abfb4c1f7f5d360969a121c66df47f65e48efd51/Model/Runner.php#L18 recommend deletion of private variable as no longer used.